src/components/SearchAutocomplete/index.js

```diff
@@ -225,3 +200,36 @@ export default class SearchAutocomplete extends Component {
     );
   }
 }
+
+
+function mapStateToProps(state) {
+  const surahs = state.surahs.entities;
+  const surahId = state.surahs.current;
+  let lang = 'en';
+
+  if (state.ayahs && state.ayahs.entities && state.ayahs.entities[surahId]) {
+    const ayahs = state.ayahs.entities[surahId];
+    const ayahKey = Object.keys(ayahs)[0];
+
+    if (ayahKey) {
+      const ayah = ayahs[ayahKey];
+
+      if (ayah.content && ayah.content[0] && ayah.content[0].lang) {
+        lang = ayah.content[0].lang;
+      }
+    }
+  }
+
+  return {
+    surahs,
+    lang
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    push: bindActionCreators(push, dispatch)
```

Comment:
`bindActionCreators` is actually not needed and is implicit when you return an object with the actions. So `{ push }` does the same as `push: bindActionCreators(push, dispatch)` no?